pkg/k8s/apis/cilium.io/client/crds/v2/ciliumclusterwidenetworkpolicies.yaml

```
@@ -1505,6 +1505,28 @@ spec:
                       type: array
                   type: object
                 type: array
+              enableDefaultDeny:
+                description: "EnableDefaultDeny configures whether this policy should
+                  configure the selected endpoint(s) to have a default deny mode.
+                  If enabled, this causes all traffic not explicitly allowed by a
+                  network policy to be dropped. \n If not specified, the default is
+                  true for each traffic direction that has rules, and false otherwise.
+                  In other words, if a policy only has Ingress or IngressDeny rules,
+                  then the default for ingress is true, and egress is false. \n If
+                  multiple policies apply to an endpoint, that endpoint's default
+                  deny will be enabled if any policy requests it. \n This is useful
+                  for creating broad-based network policies that will not cause endpoints
+                  to enter default-deny mode."
```

A couple of wording suggestions on this description, small nits. Let me know what you think:

```suggestion
                description: "EnableDefaultDeny determines whether this policy
                  configures the selected endpoint(s) to have a default deny mode.
                  If enabled, all traffic from the selected endpoint(s) not explicitly allowed by a
                  network policy will be dropped. \n If not specified, the default is
                  true for each traffic direction that has rules, and false otherwise.
                  For example, if a policy only has Ingress or IngressDeny rules,
                  then the default for ingress is true, and egress is false. \n If
                  multiple policies apply to an endpoint, that endpoint's default
                  deny will be enabled if any policy requests it. \n This is useful
                  for creating broad-based network policies that will not cause endpoints
                  to enter default-deny mode."
```

---

"from the selected endpoints" would not be correct, since this applies separately for ingress and egress. Otherwise, nice suggestions.

---

Ah good point, that makes sense.